src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { FilterOption } from '../../types/FilterOption';
+
+interface Props {
+  todos: Todo[];
+  filter: string;
+  setFilter: (filter: FilterOption) => void;
+}
+
+export const Footer: React.FC<Props> = ({ todos, filter, setFilter }) => {
+  const completedTodos = todos.filter((todo) => todo.completed);
+  const hasCompletedTodo = todos.find((todo) => todo.completed);
```

Comment:
```suggestion
  const completedTodos = todos.filter((todo) => todo.completed);
```
You can just check if completedTodos length more that zero